app/models/email_address.rb

```diff
@@ -7,6 +7,9 @@ class EmailAddress < ApplicationRecord
   validates :encrypted_email, presence: true
   validates :email_fingerprint, presence: true
 
+  scope :confirmed, -> { where('confirmed_at IS NOT NULL') }
```

Comment:
I guess technically this could have been a separate PR/change....but IMO it *really* improves readability and I wanted it in my fix